src/Engines/engine.ts

```diff
@@ -546,6 +546,8 @@ export class Engine extends ThinEngine {
 
         Engine.Instances.push(this);
 
+        (this.supportRenderPasses as any) = true;

```

Comment:
Could we avoid the any here ?

Reply:
`supportRenderPasses` is declared as `readonly` as there's no reason for the user to change the value but I needed to create this property to ease the implementation of some other code. I don't know how to change this value by keeping the `readonly` modifier?

Reply:
maybe it could be in engineFeatures ?

Reply:
Good idea!